### PR TITLE
edite Database schema

### PR DESCRIPTION
**Pull request by AhmedEhab125**
edite data base schema

### CODE REVIEW

**Comment by AhmedEhab125** [comment]
fix conflict

File: app/src/main/java/com/example/myapplication/MainActivity.kt [hunk 1]
```diff
@@ -20,6 +22,29 @@ class MainActivity : AppCompatActivity() {
         if ( UserInfoDataSource.getInstance().readFromShared(applicationContext) != null){
             println("User Aleardy Login")
         }
+        var title = "CNN"
```
Comment (ahmedMubarak2024) [inline app/src/main/java/com/example/myapplication/MainActivity.kt]: remove testing code

File: app/src/main/java/com/example/myapplication/DataBaseRepo/DataBaseRepo.kt [hunk 2]
```diff
@@ -1,21 +1,36 @@
 package com.example.myapplication.DataBaseRepo
```
Comment (ahmedMubarak2024) [inline app/src/main/java/com/example/myapplication/DataBaseRepo/DataBaseRepo.kt:1]: move to NewsRepo

File: app/src/main/java/com/example/myapplication/database/FavouriteArticlesDAtaBase.kt [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+import com.example.myapplication.database.ArticlesDao.ArticlesDao
+import com.example.myapplication.database.FavouriteDao.FavouriteDao
+import com.example.myapplication.model.Articles
+import com.example.myapplication.register.model.FavouriteArticles
+
+class FavouriteArticlesDAtaBase {
```
Comment (ahmedMubarak2024) [inline app/src/main/java/com/example/myapplication/database/FavouriteArticlesDAtaBase.kt]: remove extra database

File: app/src/main/java/com/example/myapplication/MainActivity.kt [hunk 4]
```diff
@@ -34,8 +33,7 @@ class MainActivity : AppCompatActivity() {
             title, discription, "", img, date, ""
         )
 
-        var db =DataBaseRepo(NewsDataBase.ArticlesDataBase.getInstance(this),
-            FavouriteArticlesDAtaBase.FavArticlesDataBase.getInstance(this))
+        var db =DataBaseRepo(NewsDataBase.ArticlesDataBase.getInstance(this))
         lifecycleScope.launch (Dispatchers.IO){
```
Comment (ahmedMubarak2024) [inline app/src/main/java/com/example/myapplication/MainActivity.kt]: remove test lines

File: app/src/main/java/com/example/myapplication/DataBaseRepo/DataBaseRepo.kt [hunk 5]
```diff
@@ -1,25 +1,23 @@
...
 
-import com.example.myapplication.database.DataBaseInter
-import com.example.myapplication.database.FavouriteArticlesDAtaBase
 import com.example.myapplication.database.NewsDataBase
 import com.example.myapplication.model.Articles
 import com.example.myapplication.register.model.FavouriteArticles
 
 class DataBaseRepo(var articlesDataBase: NewsDataBase.ArticlesDataBase
-, var favouriteArticlesDAtaBase: FavouriteArticlesDAtaBase.FavArticlesDataBase) : DataSourceInter {
```
Comment (ahmedMubarak2024) [inline app/src/main/java/com/example/myapplication/DataBaseRepo/DataBaseRepo.kt]: move this code to News Repo

**Review by ahmedMubarak2024 — APPROVED** [review]

**Review by ahmedMubarak2024 — APPROVED** [review]

**Pull request merged**